ci-operator/step-registry/openshift/e2e/openstack/ipi/openshift-e2e-openstack-ipi-workflow.yaml

```diff
@@ -9,11 +9,9 @@ workflow:
     - chain: ipi-openstack-post
     env:
       OPENSTACK_EXTERNAL_NETWORK: "external"
-      OS_CLOUD: "openstack-cloud"
-      OPENSTACK_COMPUTE_FLAVOR: "m1.s2.xlarge"
+      OPENSTACK_INSTANCE_FLAVOR: "m1.s2.xlarge"
       CONFIG_API_VERSION: "v1beta4"
       CONFIG_TYPE: "minimal"
-      BASE_DOMAIN: shiftstack.devcluster.openshift.com
```

Comment:
BASE_DOMAIN seems to be always the same, so removed from must workflows. Users can still add it if they want. But the default is now shiftstack.devcluster.openshift.com